documentation/modules/rn-2.4.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-24_{context}"]
+= {project-full} 2.4
+
+You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-24_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Technical change 1
+
+Description 1
+
+.Technical change 2
+
+Description 2
+
+[id="new-features-and-enhancements-24_{context}"]
+== New features and enhancements
+
+This release has the following features and improvements:
+
+.link:https://issues.redhat.com/browse/MTV-427[MTV-427] - OCP console plugin
+
+MTV Operator installed from CLI. +
+MTV web console URL.
+
+.link:https://issues.redhat.com/browse/MTV-426[MTV-426] - OpenStack migration
+
+See new link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.3/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#rhv-prerequisites_mtv[prerequisites] for RHV and VMware migration, and link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.3/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#migrating-virtual-machines-cli_mtv[procedure] to migrate virtual machines.
+
+[id="known-issues-24_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+.link:https://issues.redhat.com/browse/MTV-505[MTV-505] - Power off OpenStack VM in cold migration
+
+This issue implements the following: +
+
+` pkg/controller/plan/adapter/openstack/client.go` +
+` // Power off the source VM.` +
+` func (c *Client) PowerOff(vmRef ref.Ref) error` +
+` { return nil }`
+
+.link:https://issues.redhat.com/browse/MTV-349[MTV-349] - RHV snapshots are not deleted after a successful migration
+
+Issue description
+
+.link:https://issues.redhat.com/browse/MTV-456[MTV-456] - Migration fails during precopy/cutover while a snapshot operation is executed on the source VM
+
+Warm migration fails, returning an error, when performing another snapshot operation (create/delete) on the RHV source VM.
+Workaround: Wait for the snapshot operation to complete before running the next precopy/cutover procedure.
```

Comment:
yeah, we now know it's even broader than this after the demo :) we didn't change warm migration, so I'd keep the following two that appear in 2.3 doc:
```
Some warm migrations from RHV might fail

When you run a migration plan for warm migration of multiple VMs from RHV, the migrations of some VMs might fail during the cutover stage. In that case, restart the migration plan and set the cutover time for the VM migrations that failed in the first run. [(BZ#2063531)](https://bugzilla.redhat.com/show_bug.cgi?id=2063531)
```
```
Warm migration from RHV fails if a snapshot operation is performed on the source VM

If the user performs a snapshot operation on the source VM at the time when a migration snapshot is scheduled, the migration fails instead of waiting for the user’s snapshot operation to finish. [(BZ#2057459)](https://bugzilla.redhat.com/show_bug.cgi?id=2057459)
```
(with replace the last bug with https://issues.redhat.com/browse/MTV-456)